content/concept-amendments.md

```diff
@@ -111,7 +113,8 @@ The following is a comprehensive list of all known amendments and their status o
 
 | Name                            | Introduced | Enabled |
 |---------------------------------|------------|---------|
-| [FlowV2](#flowv2)               | v0.32.1    | Expected 2016-08-24 |
+| [Flow](#flow)                   | TBD        | TBD |
+| [FlowV2](#flowv2)               | v0.32.1    | Vetoed |
```

Comment:
@mDuo13 This goes to your point in [DOC-525](https://ripplelabs.atlassian.net/browse/DOC-525). I wonder if there's a more accurate term we could use that doesn't connote finality in the same way, i.e. suspended or deferred.